chains/types/nodle.json

```diff
@@ -0,0 +1 @@
+{"runtime_id": 1, "types": {"Balance": "u128", "Index": "u32", "Phase": "frame_system.Phase", "Address": "sp_runtime.multiaddress.MultiAddress", "ExtrinsicSignature": "sp_runtime.MultiSignature", "ParaId": "polkadot_parachain.primitives.Id", "runtime_eden.Event": "GenericEvent", "runtime_eden.Call": "GenericCall", "sp_core.crypto.AccountId32": "GenericAccountId", "pallet_identity.types.Data": "Data"}, "versioning": []}
```

Comment:
Let's make the json prettier by formatting.

Reply:
done